tests/testing.ts

```diff
@@ -403,13 +215,90 @@ function checkNoJson(path: string): boolean {
   );
 }
 
-function postTestCleanup(): void {
+export function postTestCleanup(warpTestFolder: string): void {
   deleteJson('warplib');
-  fs.rmSync(WARP_TEST_FOLDER, { recursive: true, force: true });
+  fs.rmSync(warpTestFolder, { recursive: true, force: true });
 }
 
 function deleteJson(path: string): void {
   findAllFiles(path, true)
     .filter((file) => file.endsWith('.json'))
     .forEach((file) => fs.unlinkSync(file));
 }
+
+export function runTests(
+  expectedResults: Map<string, ResultType>,
+  warpTest: string,
+  warpTestFolder: string,
+  warpCompilationTestPath: string,
+  contractsFolder: string,
+) {
+  describe('Running compilation tests', function () {
+    this.timeout(TIME_LIMIT);
+
+    let onlyResults: boolean, unsafe: boolean, force: boolean, exact: boolean;
+    let filter: string | undefined;
+
+    const results = new Map<string, ResultType>();
+
+    this.beforeAll(() => {
+      onlyResults = process.argv.includes('--only-results') ? true : false;
+      unsafe = process.argv.includes('--unsafe') ? true : false;
+      force = process.argv.includes('--force') ? true : false;
+      exact = process.argv.includes('--exact') ? true : false;
```

Comment:
Isn't the result of `includes` already boolean?

Reply:
Good observation!